x-pack/plugins/observability_solution/observability_ai_assistant_app/scripts/evaluation/evaluation.ts

```diff
@@ -275,7 +275,45 @@ function runEvaluations() {
               );
 
             log.write('-------------------------------------------');
-            log.write(`Scored ${score.score} out of ${score.total}`);
+            log.write(
+              `Model ${connector.id} scored ${modelScore.score} out of ${modelScore.total}`
+            );
+            log.write('-------------------------------------------');
+
+            const scoresByScenario: {
+              [key: string]: {
+                score: number;
+                total: number;
+              };
+            } = results.reduce(
+              (
+                acc: {
+                  [key: string]: {
+                    score: number;
+                    total: number;
+                  };
+                },
+                result
+              ) => {
+                const scenario = result.name.trim().split(' ')[0];
```

Comment:
this kinda breaks on spaces in the title no? I think we can add a `category` property on the EvaluationResult by using the title of the first suite here:  https://github.com/elastic/kibana/blob/551e1232b870654d036aff4d52f43443ee3ecf20/x-pack/plugins/observability_solution/observability_ai_assistant_app/scripts/evaluation/kibana_client.ts#L203-L213

Reply:
yes, you're 100% right, fixed to use the suite name as the category
<img width="291" alt="Screenshot 2024-05-24 at 16 48 35" src="https://github.com/elastic/kibana/assets/2208394/46944f1d-7bd7-4e57-925d-a54c2c370e91">